src/effects/effectmanifestparameter.h

```diff
@@ -40,11 +40,17 @@ class EffectManifestParameter {
         NUM_LINK_TYPES
     };
 
+    enum LinkInversion {
```

Comment:
Well you don't need to go through the whole file to fix the new ones... Plain enums are just [bad](http://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum).